server.js

```diff
@@ -3,35 +3,99 @@ import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
+import dotenv from 'dotenv'
+import listEndpoints from 'express-list-endpoints'
+
+import data from './data/golden-globes.json'
+
+dotenv.config()
+
+const mongoUrl = process.env.MONGO_URL || `mongodb+srv://${process.env.USER_ID}:${process.env.API_KEY}@cluster0.bvwog.mongodb.net/goldenGlobe?retryWrites=true&w=majority`
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
+const goldenGlobeSchema = new mongoose.Schema({
+  year_film: Number,
+  year_award: Number,
+  ceremony: Number,
+  category: String,
+  nominee: String,
+  film: String,
+  win: Boolean
+})
+
+const GoldenGlobe = mongoose.model('GoldenGlobe', goldenGlobeSchema)
+
+if(process.env.RESET_DB) {
+  console.log('SEEDING')
+  const seedDB = async () => {
+    await GoldenGlobe.deleteMany()
+
+    await data.forEach(item => {  
+      const newGoldenGlobe = new GoldenGlobe(item)
+      newGoldenGlobe.save() 
+    })
+  }
+  seedDB()
+}
+
 const port = process.env.PORT || 8080
 const app = express()
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors())
-app.use(bodyParser.json())
+app.use(express.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  response.send(listEndpoints(app))
+})
+app.get('/goldenglobes/film', async(req, res) => {
```

Comment:
when you want to use query to filter, then do not create a new endpoint. Add your filtering to the main GET endpoint for this dataset. `/goldenglobes?film=blablabla` not `/goldenglobes/film?film=blablabla`